packages/alchemy/src/config/actions/createAccount.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  createLightAccount,
+  createMultiOwnerModularAccount,
+  type CreateLightAccountParams,
+  type CreateMultiOwnerModularAccountParams,
+} from "@alchemy/aa-accounts";
+import { custom } from "viem";
+import { AlchemySignerStatus } from "../../signer/index.js";
+import { ClientOnlyPropertyError } from "../errors.js";
+import type {
+  AlchemyAccountsConfig,
+  SupportedAccountTypes,
+  SupportedAccounts,
+} from "../types";
+import { getSignerStatus } from "./getSignerStatus.js";
+
+export type AccountConfig<TAccount extends SupportedAccountTypes> =
+  TAccount extends "LightAccount"
+    ? Omit<CreateLightAccountParams, "signer" | "transport" | "chain">
+    : Omit<
+        CreateMultiOwnerModularAccountParams,
+        "signer" | "transport" | "chain"
```

Comment:
actually, we shouldn't omit the `signer` here and instead make it optional. that would allow someone to create a session key signer based account

Reply:
not gonna do this now actually as it requires customizing signer in a lot of other places. opting for simplicity over flexibility / power in this first approach

Reply:
if we want session keys, we can always add a hook for that later